src/web/crate_details.rs

```diff
@@ -237,6 +266,13 @@ impl CrateDetails {
         Ok(Some(crate_details))
     }
 
+    fn enrich_readme(&mut self, storage: &Storage) -> Result<()> {
```

Comment:
nit: shouldn't this be just `fetch_readme`, returning the readme?

Reply:
Merged this up into the caller, and moved the `fetch_readme_from_storage` into an associated method.